### DISCUSSION TITLE
Are custom snippets possible for a single session?

### DISCUSSION THREAD

**Discussion by medwatt** [L3MON4D3/LuaSnip#1004] [Q&A] [answered]
Let's say I have a generic snippet defined as `\mintinline{$1}{$2}`. Lets say I am currently writing a latex document on some `language`. I find myself having to replace `$1` with `language`  every time. I would rather have the snippet expand to `\mintinline{language}{$2}` in this specific session. I wonder if there's some magic in luasnip that can do this?

**Comment by L3MON4D3** [comment, marked as answer]
Would making snippets local to some project work? You can call `require("luasnip.loaders.from_lua").load({paths = {vim.fn.getcwd() .. "/.luasnippets/"}})` to load from a `.luasnippets`-directory in the current project (well, requires you to start nvim from the root-dir of that project, but for me at least that's always the case).

**Reply by medwatt** [reply]
Perfect!
